packages/common/src/ui/layout/tables/columns/EditableQuantityColumn.tsx

```diff
@@ -0,0 +1,73 @@
+import { ColumnDefinition } from './types';
+import { DomainObject } from '../../../../types';
+import { TextField, InputAdornment, Tooltip, Typography } from '@mui/material';
+import React, { useState, useEffect } from 'react';
+
+interface SomeQuantityEntity extends DomainObject {
+  quantity: number;
+  setQuantity: (rowKey: number, newQuantity: number) => void;
+}
+
+export const getEditableQuantityColumn = <
```

Comment:
didn't touch the styles, just wanted to get the pattern for editable cells in place

Reply:
Yup. I also wasn't exactly sure what to do with the styles as there's no examples other than the add item dialog, but the styling on that is also a slightly different